js/adapt-tabs.js

```diff
@@ -51,6 +52,21 @@ define(function(require) {
 			});
 		},
 
+		setTabText: function() {
+			var items = this.model.get('_items');
+			if (Adapt.device.screenSize == 'small') {
```

Comment:
Can I suggest the following to replace this `if` statement? Not only is it DRYer but it allows for the standard tab title to be used as a fallback on mobile if the mobileTitle hasn't been specified.
```
var items = this.model.get('_items');
_.each(items, function (item, index) {
	var tabTitle = (Adapt.device.screenSize === 'small' && item.mobileTabTitle) ? item.mobileTabTitle : item.tabTitle;
	this.$('.tabs-navigation-item-inner').eq(index).html(tabTitle);
});
```